src/app/lib/config/services/ws.ts

```diff
@@ -18,8 +18,8 @@ export const service: DefaultServiceConfig = {
     serviceLocalizedName: 'World Service',
     product: 'BBC News',
     defaultImage:
-      'https://static.files.bbci.co.uk/ws/simorgh-assets/public/news/images/metadata/poster-1024x576.png',
-    defaultImageAltText: 'BBC News',
+      'https://static.files.bbci.co.uk/ws/simorgh-assets/public/news/images/metadata/poster-1200x630.png',
+    defaultImageAltText: 'BBC World Service Image',
```

Comment:
It's not best practice to include the word "image" in alt text. 
```suggestion
    defaultImageAltText: 'BBC News World Service',
```